etc/profile.d/git-sdk.sh

```diff
@@ -113,7 +113,9 @@ sdk () {
 		if test git = "$2" && test ! -f "$src_dir/config.mak"
 		then
 			cat >"$src_dir/config.mak" <<-\EOF
+			ifndef MSVC
 			DEVELOPER=1
```

Comment:
There was a recent discussion on the git mailing list where it was discussed whether DEVELOPER=1 should be set by default.  The decision was that it should not be set.  For this patch, I'd just remove the DEVELOPER=1 line entirely so we are consistent with that decision.

https://public-inbox.org/git/20180804020009.224582-1-sbeller@google.com/

Getting the MSVC compiler to work when DEVELOPER=1 is a separate task/work item.  Given with git (unlike many other projects) DEVELOPER=1 is only setting gcc specific warnings and not turning on additional run time test, etc; I don't think it makes sense to enable that for MSVC builds (trying to map gcc specific warnings to cl specific warnings would be an "interesting" challenge).